app/controllers/mobile.js

```diff
@@ -0,0 +1,3 @@
+import MobileRoute from '../routes/mobile';
+
+export default class MobileController extends MobileRoute {}
```

Comment:
Why controller is extending MobileRoute here? I think it should be `Controller`. Check other controller files for reference

Reply:
replaced